base/special/trig.jl

```diff
@@ -1063,31 +1063,86 @@ isinf_real(x::Complex) = isinf(real(x)) && isfinite(imag(x))
 isinf_real(x::Number) = false
 
 """
-    sinc(x)
+    sinc(x; [normalized::Bool=true])
+
+Compute ``\\sin(\\pi x) / (\\pi x)`` if ``x \\neq 0``, and ``1`` if ``x = 0`` (equal to the
+[`sincpi`](@ref) function).  If `normalized` is `false`, the factors of ``\\pi`` are
+omitted (equal to the [`sincu`](@ref) function).
+
+The normalization is with respect to integration over all real values of ``x``:
+```math
+\\int_{-\\infty}^{\\infty} \\mathrm{sincpi}(x) \\, \\mathrm{d}x = 1,
```

Comment:
```suggestion
\\int_{-\\infty}^{\\infty} \\operatorname{sincpi}(x) \\, \\operatorname{d}x = 1,
```
It seems `\operatorname` is better here.
https://tex.stackexchange.com/questions/48459/whats-the-difference-between-mathrm-and-operatorname

Reply:
At least for `sincpi`, I agree that it would be in principle, but I shy away from it because this presumably needs to be rendered with MathJax, which [apparently requires the `ams` package](https://docs.mathjax.org/en/latest/input/tex/macros/index.html#o) to be loaded, and I'm not sure that is likely to be the case wherever the docs would be hosted.  As for `d`, I think it would leave an unwanted space before the `x`.

But in any case, I think this is all moot, since this PR is seemingly destined for the rubbish bin (at least in this repo).

Reply:
>this presumably needs to be rendered with MathJax, which [apparently requires the ams package](https://docs.mathjax.org/en/latest/input/tex/macros/index.html#o) to be loaded, and I'm not sure that is likely to be the case wherever the docs would be hosted.

It seems KaTeX is used in Documenter.jl, and `\operatorname` is used in other places. See [`eps` docstring](https://github.com/JuliaLang/julia/blob/bed2cd540a11544ed4be381d471bbf590f0b745e/base/float.jl#L1051) and [its rendered documentation](https://docs.julialang.org/en/v1/base/base/#Base.eps-Tuple{AbstractFloat}).

>As for `d`, I think it would leave an unwanted space before the `x`.

Sorry, that was my mistake.

>But in any case, I think this is all moot, since this PR is seemingly destined for the rubbish bin (at least in this repo).

I see. I think adding `sinc` and `sincpi` functions (https://github.com/JuliaLang/julia/issues/36954#issuecomment-1296954308) to SpecialFunctions.jl would be a better solution.